nucliadb_sdk/nucliadb_sdk/client.py

```diff
@@ -81,25 +87,25 @@ def __init__(
     ):
         self.api_key = api_key
         self.environment = environment
-        self.url = url
+        # self.url = url
 
         internal_hosts_set = all((writer_host, reader_host, search_host, train_host))
         url_set = bool(url)
 
-        if not (url_set or internal_hosts_set):
-            raise AttributeError("Either url or nucliadb services hosts must be set")
-
-        if url_set:
+        if url_set and url is not None:
             self.url = url
-        elif internal_hosts_set:
+        elif internal_hosts_set and writer_host is not None:
```

Comment:
I think this is redundant, as `writer_host` is being checked in `internal_hosts_set`.

So we were good with `elif internal_hosts_set`

Reply:
Its for typing